index.html

```diff
@@ -202,7 +202,8 @@ <h3 id="allpeople" data-export="" data-dfn-type="dfn">
 People should not need a high level of technical literacy to use the web.
 Web platform technologies should behave consistently and intuitively.
 We will build internationalization and localization capabilities
-into our specifications and websites.
+into our specifications and websites, including support for different languages.
+Our work should not disadvantage smaller language or cultural communities.
```

Comment:
```suggestion
Our work should not disadvantage smaller cultural communities
or people who speak minority languages.
```

Reply:
@torgo asked to clarify this sentence. @aphillips Could you let me know if "minority languages" is the right phrase to use here, or if there's a better term? Thanks.

Reply:
Some of my colleagues have cited the term "minority language" as a pejorative, while others use it as a convenience. A challenge here is that not all disadvantaged languages are small. Non-official languages, oppressed cultures, etc. sometimes can be quite large. 

The pedantic part of me wants to quibble with the front part of the sentence too. We don't build "internationalization capabilities" for the most part: we build technologies that _are_ internationalized 😄. Drawing upon [this](https://www.w3.org/International/i18n-drafts/nav/about#i18n-activity), perhaps:

```suggestion
Our specifications and websites are well internationalized,
provide support for language and cultural adaptation, and support localization,
so that our work is accessible to all users, regardless of language, writing system,
or culture.
```